src/lasdi/latent_space.py

```diff
@@ -1,175 +1,480 @@
-            'PReLU': torch.nn.PReLU,
-            'ReLU': torch.nn.ReLU,
-            'ReLU6': torch.nn.ReLU6,
-            'RReLU': torch.nn.RReLU,
-            'SELU': torch.nn.SELU,
-            'CELU': torch.nn.CELU,
-            'GELU': torch.nn.GELU,
-            'sigmoid': torch.nn.Sigmoid,
-            'SiLU': torch.nn.SiLU,
-            'mish': torch.nn.Mish,
-            'softplus': torch.nn.Softplus,
-            'softshrink': torch.nn.Softshrink,
-            'tanh': torch.nn.Tanh,
-            'tanhshrink': torch.nn.Tanhshrink,
-            'threshold': torch.nn.Threshold,
-            }
-
-def initial_condition_latent(param_grid, physics, autoencoder):
-
-    '''
-
-    Outputs the initial condition in the latent space: Z0 = encoder(U0)
-
-    '''
-
-    n_param = param_grid.shape[0]
-    Z0 = []
-
-    sol_shape = [1, 1] + physics.qgrid_size
+act_dict = {'ELU'           : torch.nn.ELU,
+            'hardshrink'    : torch.nn.Hardshrink,
+            'hardsigmoid'   : torch.nn.Hardsigmoid,
+            'hardtanh'      : torch.nn.Hardtanh,
+            'hardswish'     : torch.nn.Hardswish,
+            'leakyReLU'     : torch.nn.LeakyReLU,
+            'logsigmoid'    : torch.nn.LogSigmoid,
+            'PReLU'         : torch.nn.PReLU,
+            'ReLU'          : torch.nn.ReLU,
+            'ReLU6'         : torch.nn.ReLU6,
+            'RReLU'         : torch.nn.RReLU,
+            'SELU'          : torch.nn.SELU,
+            'CELU'          : torch.nn.CELU,
+            'GELU'          : torch.nn.GELU,
+            'sigmoid'       : torch.nn.Sigmoid,
+            'SiLU'          : torch.nn.SiLU,
+            'mish'          : torch.nn.Mish,
+            'softplus'      : torch.nn.Softplus,
+            'softshrink'    : torch.nn.Softshrink,
+            'tanh'          : torch.nn.Tanh,
+            'tanhshrink'    : torch.nn.Tanhshrink,
+            'threshold'     : torch.nn.Threshold,}
+
+
+
+# -------------------------------------------------------------------------------------------------
+# initial_conditions_latent function
+# -------------------------------------------------------------------------------------------------
+
+def initial_condition_latent(param_grid     : np.ndarray, 
+                             physics        : Physics, 
+                             autoencoder    : torch.nn.Module) -> list[np.ndarray]:
+    """
+    This function maps a set of initial conditions for the fom to initial conditions for the 
+    latent space dynamics. Specifically, we take in a set of possible parameter values. For each 
+    set of parameter values, we recover the fom IC (from physics), then map this fom IC to a 
+    latent space IC (by encoding it using the autoencoder). We do this for each parameter 
+    combination and then return a list housing the latent space ICs.
+
+    
+    -----------------------------------------------------------------------------------------------
```

Comment:
I think this format is not recognized by sphinx and makes param_grid description hidden in the API website (see this [webpage](https://software.llnl.gov/GPLaSDI/autoapi/lasdi/latent_space/index.html)).

Changing line 51-53 to `Args:` and taking an indent of line 55-60 will resolve the issue.